core-tests/shared/src/test/scala/zio/metrics/MetricSpec.scala

```diff
@@ -223,6 +223,16 @@ object MetricSpec extends ZIOBaseSpec {
           r1 <- base.tagged(MetricLabel("dyn", "x")).value
           r2 <- base.tagged(MetricLabel("dyn", "xyz")).value
         } yield assertTrue(r0.count == 0L, r1.count == 1L, r2.count == 1L)
+      },
+      test("linear boundaries") {
```

Comment:
Turns out linear boundaries are good as is, but I've already written a test and decided not to remove it